backend/assessments/1/src/main/java/org/example/Choice.java

```diff
@@ -0,0 +1,59 @@
+package org.example;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+
+public class Choice {
+    public static List<String> getBowlers40(List<Player> players, String team){
+        return players.stream()
+                .filter(p -> p.getTeam().equalsIgnoreCase(team))
+                .filter(p -> p.getWickets() >= 40)
+                .map(Player::getName)
+                .toList();
+    }
+
+    public static void highest(List<Player> players, String team){
+        String wicketTaker= String.valueOf(highestWicket(players, team));
+        String runMaker= String.valueOf(highestRun(players, team));
+        System.out.println("Highest wicket taker is " + wicketTaker);
+        System.out.println("Highest run maker is " + runMaker);
+    }
+
+    private static Optional<String> highestWicket(List<Player> players, String team){
+        return players.stream()
+                .filter(p -> p.getTeam().equalsIgnoreCase(team))
+                .max((p1, p2) -> p1.getWickets() - p2.getWickets())
+                .map(Player::getName);
```

Comment:
You did it correctly
Then why did you map on name should have returned complete info and logged it